test/e2e/framework/pod/utils.go

```diff
@@ -304,12 +304,12 @@ func VerifyCgroupValue(f *framework.Framework, pod *v1.Pod, cName, cgPath string
 		pod.Namespace, pod.Name, cName, expectedCgValues, cgPath)
 	cgValue, _, err := ExecCommandInContainerWithFullOutput(f, pod.Name, cName, "/bin/sh", "-c", cmd)
 	if err != nil {
-		return fmt.Errorf("failed to find one of the expected cgroup values %q in container cgroup %q", expectedCgValues, cgPath)
+		return fmt.Errorf("failed to find one of the expected cgroup values %q in container cgroup %q: %w", expectedCgValues, cgPath, err)
```

Comment:
This error isn't really an issue with the expectations not being matched - it's just that the command get the actual values failed, right?

```suggestion
		return fmt.Errorf("failed to read cgroup %q: %w", cgPath, err)
```

Reply:
yeah, updated the error message